### PR TITLE
Update Arkansas parameters based on 2023 tax form

### LINKED ISSUE #3804: Update Arkansas parameters based on 2023 tax form
Update on Arkansas parameter

### PR DESCRIPTION
**Pull request by Kiki-67**
Fixes #3804

### CODE REVIEW

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: need a changelog

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/start.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/start.yaml:1]: add references to all yaml files

File: policyengine_us/parameters/gov/states/ar/tax/income/rates/main/reduction.yaml [hunk 2]
```diff
@@ -20,309 +20,411 @@ brackets:
...
       2021-01-01: .inf
       2022-01-01: 91_801
+      2023-01-01: .inf
     amount:
       2021-01-01: 247.5
       2022-01-01: 167.20
+      2023-01-01: 153.70
   - threshold:
       2021-01-01: .inf
       2022-01-01: .inf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/rates/main/reduction.yaml:430]: might need to add a .inf bracket to the bottom -> all yaml files

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/ar/tax/income/rates/main/rate.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/rates/main/rate.yaml:1]: add one test for main rate, for the maximum bracket

File: changelog_entry.yaml (hunk 1 above)
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: add changelog

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/start.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/start.yaml:1]: add the 2023 form as a reference to all yaml files

File: policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/head_of_household/two_or_more_dependents.yaml [hunk 3]
```diff
@@ -19,224 +21,312 @@ brackets:
...
       2021-01-01: 26_000
       2022-01-01: 26_600
+      2023-01-01: 27_300
     amount:
       2021-01-01: .inf
+      2023-01-01: 574
+  - threshold:
+      2023-01-01: 27_400
+    amount:
+      2023-01-01: .inf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/head_of_household/two_or_more_dependents.yaml:328]:
```suggestion
      2021-01-01: .inf      
```

File: policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/head_of_household/two_or_more_dependents.yaml [hunk 4]
```diff
@@ -19,224 +21,312 @@ brackets:
...
+      2023-01-01: 27_300
     amount:
       2021-01-01: .inf
+      2023-01-01: 574
+  - threshold:
+      2023-01-01: 27_400
+    amount:
+      2023-01-01: .inf      
+  - threshold:
+      2023-01-01: .inf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/head_of_household/two_or_more_dependents.yaml:330]:
```suggestion
      2021-01-01: .inf
```

File: policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/head_of_household/two_or_more_dependents.yaml [hunk 5]
```diff
@@ -19,224 +21,312 @@ brackets:
...
+    amount:
+      2023-01-01: .inf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/head_of_household/two_or_more_dependents.yaml:332]:
```suggestion
      2021-01-01: .inf
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/single.yaml [hunk 6]
```diff
@@ -19,168 +21,228 @@ brackets:
...
       2022-01-01: .inf
+      2023-01-01: 209
   - threshold:
       2021-01-01: 15_700
       2022-01-01: .inf
+      2023-01-01: 16_500
     amount:
       2021-01-01: .inf
       2022-01-01: .inf
+      2023-01-01: .inf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/single.yaml:244]:
```suggestion
```

File: policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/single.yaml [hunk 7]
```diff
@@ -19,168 +21,228 @@ brackets:
...
+  - threshold:
+      2021-01-01: .inf
+    amount:
+      2021-01-01: .inf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/single.yaml:248]:
```suggestion
```

File: policyengine_us/parameters/gov/states/ar/tax/income/rates/main/reduction.yaml [hunk 8]
```diff
@@ -20,312 +21,420 @@ brackets:
...
+      2023-01-01: .inf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/rates/main/reduction.yaml:432]:
```suggestion
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ar/tax/income/rates/main/reduction.yaml:432]: same adj. for all of rates files

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/ar/tax/income/deductions/itemized/tuition/weighted_average_tuition/two_year_college.yaml [hunk 9]
```diff
@@ -2,7 +2,7 @@ description: Arkansas defines the weighted average tuition for four-year college
 values:
   2021-01-01: 2_141
   2022-01-01: 2_272
-
+  2023-01-01: 4_405
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/deductions/itemized/tuition/weighted_average_tuition/two_year_college.yaml:5]: shouldn't this be [this](https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/AR1075_2023.pdf#page=1) number:

image

File: policyengine_us/parameters/gov/states/ar/tax/income/rates/main/reduction.yaml [hunk 10]
```diff
@@ -20,312 +21,418 @@ brackets:
...
       2022-01-01: 91_601
+      2023-01-01: 94_201
     amount:
       2021-01-01: 247.5
       2022-01-01: 187.20
+      2023-01-01: 153.70
   - threshold:
       2021-01-01: .inf
       2022-01-01: 91_701
+      2023-01-01: 100_001
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/rates/main/reduction.yaml:416]:
```suggestion
      2023-01-01: .inf
```

File: policyengine_us/parameters/gov/states/ar/tax/income/rates/main/reduction.yaml [hunk 11]
```diff
@@ -20,312 +21,418 @@ brackets:
...
       2022-01-01: 177.20
+      2023-01-01: 153.70
   - threshold:
       2021-01-01: .inf
       2022-01-01: 91_801
+      2023-01-01: .inf
     amount:
       2021-01-01: 247.5
       2022-01-01: 167.20
+      2023-01-01: 153.70
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/rates/main/reduction.yaml:428]:
```suggestion
      2023-01-01: .inf
```

File: policyengine_us/parameters/gov/states/ar/tax/income/rates/main/rate.yaml [hunk 12]
```diff
@@ -21,27 +23,37 @@ brackets:
...
       2021-01-01: 39_700
       2022-01-01: .inf
     amount:
       2021-01-01: 0.059
       2022-01-01: 0.049
+      2023-01-01: 0.047
+  - threshold:
+      2021-01-01: .inf
+    amount:
+      2021-01-01: .inf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/rates/main/rate.yaml:59]:
```suggestion
```

File: policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/joint/no_or_one_dependent.yaml [hunk 13]
```diff
@@ -19,251 +21,343 @@ brackets:
...
       2021-01-01: .inf
+      2023-01-01: 499
+  - threshold:
+      2023-01-01: 27_500
+    amount:
+      2023-01-01: .inf 
+  - threshold:
+      2021-01-01: .inf
+    amount:
+      2021-01-01: .inf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/joint/no_or_one_dependent.yaml:363]:
```suggestion
```

File: policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/head_of_household/two_or_more_dependents.yaml [hunk 14]
```diff
@@ -19,224 +21,312 @@ brackets:
...
+      2023-01-01: 561
   - threshold:
       2021-01-01: 26_000
       2022-01-01: 26_600
+      2023-01-01: 27_300
+    amount:
+      2021-01-01: .inf
+      2023-01-01: 574
+  - threshold:
+      2023-01-01: 27_400
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/rates/low_income_tax_tables/head_of_household/two_or_more_dependents.yaml:326]: add 2021 values to this if we need these brackets

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Comment by PavelMakarchuk** [comment]
Could you write out all the programs that changed in 2023 in the issue #3804 - it makes reviewing changes easier

**Comment by Kiki-67** [comment]
parameters changed in 2023:
1. cdcc
2. deductions - weighted average tuition
  2.1 four year college
  2.2 two year college
3. low income tax tables
  3.1 head of household - no or one dependent
  3.2 head of household - two or more dependent
  3.3 joint - no or one dependent
  3.4 joint - two or more dependent
  3.5 single
4. main rate 
5. main reduction

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]
